operator/apis/machinelearning.seldon.io/v1/seldondeployment_types.go

```diff
@@ -34,15 +34,15 @@ const (
 	Label_svc_orch           = "seldon-deployment-contains-svcorch"
 	Label_app                = "app"
 	Label_fluentd            = "fluentd"
-	Label_router             = "router"
-	Label_combiner           = "combiner"
-	Label_model              = "model"
-	Label_transformer        = "transformer"
-	Label_output_transformer = "output-transformer"
-	Label_default            = "default"
-	Label_shadow             = "shadow"
-	Label_canary             = "canary"
-	Label_explainer          = "explainer"
+	Label_router             = "seldon.io/router"
```

Comment:
Nice one! Namespacing the labels under `seldon.io/*` seems like the right thing to do :+1: